src/util/macro_logger.rs

```diff
@@ -0,0 +1,43 @@
+use util::logger::Record;
+
+macro_rules! log_internal {
+	($self: ident, $lvl:expr, $($arg:tt)+) => (
+		&$self.logger.log(&Record::new($lvl, format_args!($($arg)+), module_path!(), file!(), line!()));
+	);
+	($self: ident, $lvl:expr, $($arg:tt)+) => (log_internal!($self, $lvl, $($arg)+))
```

Comment:
Looks like this comment wasn't addressed: https://github.com/rust-bitcoin/rust-lightning/pull/91#discussion_r209969518